tests/eclient/testdata/ctrl_cert_change.txt

```diff
@@ -0,0 +1,70 @@
+# Test of controller certificate change
```

Comment:
It would be nice to write somewhere in comment here what exactly you check, some summary. E.g. validate re-encryption of app's userdata after controller certificate change with edge node reboot (with additional notes about 3 apps).
Additional question: can we somehow validate that we really change certificate from EVE-OS logs/info? I mean if we will put some kind of noop inside `eden adam upload-cert` (occasionally) looks like the test will pass.

Reply:
Yes, we can check `/persist/certs/server-signing-cert.pem` in EVE. Added that to the test now together with the test description.